cli/sawtooth_cli/config.py

```diff
@@ -366,6 +396,46 @@ def _do_config_list(args):
         raise AssertionError('Unknown format {}'.format(args.format))
 
 
+def _do_config_genesis(args):
+    pubkey, signing_key = _read_signing_keys(args.key)
+
+    authorized_keys = args.authorized_key if args.authorized_key else [pubkey]
+    if pubkey not in authorized_keys:
+        authorized_keys.append(pubkey)
+
+    txns = []
+
+    txns.append(_create_propose_txn(
+        pubkey, signing_key,
+        ('sawtooth.config.vote.authorized_keys',
+         ','.join(authorized_keys))))
+
+    if args.approval_threshold is not None:
+        if args.approval_threshold < 1:
+            raise CliException('approval threshold must be greater than 0')
+
+        if args.approval_threshold > len(authorized_keys):
+            raise CliException(
+                'approval_threshold must not be greater than the number of '
```

Comment:
approval threshold or approval_threshold? It should be consistent between the two CliException above